src/iadmin.cpp

```diff
@@ -2516,6 +2660,40 @@ usage( char *subOpt ) {
         ""
     };
 
+    char* set_pam_password_config_usage[] = {
+        "set_pam_password_config OPTION_NAME OPTION_VALUE",
+        " ",
+        ("Set the specified OPTION_NAME to OPTION_VALUE for pam_password configuration in the local zone in "
+         "R_GRID_CONFIGURATION."),
+        " ",
+        // TODO: Add descriptions for these option names...
+        "The list of accepted OPTION_NAMEs is:",
+        " - password_length",
+        " - no_extend",
+        " - password_min_time",
+        " - password_max_time",
```

Comment:
Should we sort these? Hmm ...

Reply:
Seeing as how we will be updating the docs anyway, we can sort them in all the places

Reply:
let's sort.   it's the most defensible position.

Reply:
Done